### PR TITLE
Improvements to build process and support for debian packages

### PR DESCRIPTION
**Pull request by amarekano**
Update to setup.py should now allow support for newer versions of pyopenssl, protobuf and pyyaml.

### CODE REVIEW

**Comment by amarekano** [comment]
Added missing deliverables to the egg being built.

**Comment by HenryHoggard** [comment]
These are failing build tests. Can you fix and resubmit.

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import os
 import setuptools
 
-from src.drozer import meta
+from drozer import meta
```
Comment (HenryHoggard) [inline setup.py]: Any reason why this has been changed? This was changed to src.drozer so you no longer need to set environment variables so it negated a lot of the bug reports.

File: setup.py [hunk 2]
```diff
@@ -49,6 +49,7 @@ def find_libs(src):
                               "lib/*.pk8",
                               "lib/weasel/armeabi/w",
                               "server/web_root/*" ] },
-  scripts = ["bin/drozer", "bin/drozer-complete"],
-  install_requires = ["protobuf==2.6.1","pyopenssl==16.2", "pyyaml==3.11"],
+  scripts = ["bin/drozer", "bin/drozer-complete", "bin/drozer-repository"],
+  install_requires = ["protobuf>=2.6.1","pyopenssl>=16.2", "pyyaml>=3.11"],
+  data_files = [('/etc/bash_completion.d',['scripts/drozer'])],
```
Comment (HenryHoggard) [inline setup.py]: I think this is the line that is breaking the build process.

File: stdeb.cfg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[DEFAULT]
+Package: drozer
+XS-Python-Version: >= 2.7
```
Comment (HenryHoggard) [inline stdeb.cfg]: Drozer works on 2.6. Don't need to force 2.7

File: README.md [hunk 4]
```diff
@@ -33,6 +33,7 @@ drozer is open source software, maintained by MWR InfoSecurity, and can be downl
 git clone https://github.com/mwrlabs/drozer/
 cd drozer
 make apks
+source ENVIRONMENT
```
Comment (HenryHoggard) [inline README.md]: See below comment regarding `src.drozer`

File: setup.py (hunk 1 above)
Comment (amarekano) [inline setup.py]: This change was required to enable stdeb to find the right package structure when building deb packages.

File: stdeb.cfg (hunk 3 above)
Comment (amarekano) [inline stdeb.cfg]: 2.6 is beyond it's EOL and we shouldn't be supporting it any longer.

File: README.md (hunk 4 above)
Comment (amarekano) [inline README.md]: as mentioned in my previous comment, this needs to be enabled to help stdeb find the right package structure.

File: setup.py (hunk 2 above)
Comment (amarekano) [inline setup.py]: I think we should support a minimum version of these dependencies instead of a specific version. Thoughts?

**Comment by HenryHoggard** [comment]
If you are able to submit each feature/fix as a separate PR that would be great. :)

File: setup.py (hunk 2 above)
Comment (HenryHoggard) [inline setup.py]: Not sure why its highlighted line 53. The line In question is 54.

**Comment by amarekano** [comment]
Can do. I'll close this PR.

**Pull request closed without merging**